Unix/tools/allocdump

```diff
@@ -0,0 +1,32 @@
+#!/bin/sh
+#
+# Script to run to collect memory allocation information for all running OMI processes
+# Note that in order for this to work:
+#   1.  Omi processes must have been built with debug turned on
+#   2.  Omi loglevel must be INFO or higher
```

Comment:
You can verify this. Can you do so in this script please? If it's not INFO or higher, then print an error and abort rather than just "doing the wrong thing".